docs/camera_api.rst

```diff
@@ -21,11 +21,11 @@ The previewOff() function will turn off the native camera preview::
 
     Bebo.previewOff();
 
-Bebo.Camera.capture
+Bebo.Camera.capturePhoto
```

Comment:
@theonlyjohnny can you add in more info here, 
what do i need to do to get the captured photo.